app/src/format.py

```diff
@@ -86,5 +66,40 @@ def format_bem_documents(
     documents = _get_bem_documents_to_show(
         docs_shown_max_num, docs_shown_min_score, chunks_with_scores
     )
-    formatted_documents = "".join([f"<li>{document.name}</li>" for document in documents])
-    return f"<h3>Source(s)</h3><ul>{formatted_documents}</ul>"
+    pdf_chunks = ""
+    for chunk_with_score in documents:
+        document = chunk_with_score.document
+        if chunk_with_score.max_score < docs_shown_min_score:
+            logger.info(
+                "Skipping chunk with score less than %f: %s",
+                docs_shown_min_score,
+                document.name,
+            )
+            continue
+        pdf_chunks += format_to_accordion_html(document=document, score=chunk_with_score.max_score)
+
+    return "<h3>Source(s)</h3>" + pdf_chunks
+
+
+def format_to_accordion_html(document: Document, score: float) -> str:
+    global _accordion_id
+    _accordion_id += 1
+    similarity_score = f"<p>Similarity Score: {str(score)}</p>"
+
+    return f"""
+    <div class="usa-accordion" id=accordion-{_accordion_id}>
+        <h4 class="usa-accordion__heading">
+            <button
+                type="button"
+                class="usa-accordion__button"
+                aria-expanded="false"
+                aria-controls="a-{_accordion_id}"
+                >
+                <a href='https://link/to/guru_card'>{document.name}</a>
+            </button>
+        </h4>
+        <div id="a-{_accordion_id}" class="usa-accordion__content usa-prose" hidden>
+            <p>Summary: {document.content.strip() if document.content else ""}</p>
```

Comment:
If `not document.content`, then we probably don't want to show `Summary: `, right?